packages/wdio-sync/webdriverio-core.d.ts

```diff
@@ -680,9 +702,11 @@ declare namespace WebdriverIO {
 
         /**
          * Retrieve a [cookie](https://w3c.github.io/webdriver/webdriver-spec.html#cookies)
+         * visible to the current page. You can query a specific cookie by providing the cookie name or
+         * retrieve all.
          */
         getCookies(
```

Comment:
I am missing the deleteCookies command

Reply:
right here: https://github.com/webdriverio/webdriverio/blob/master/packages/wdio-sync/webdriverio-core.d.ts#L674-L679